app/src/main/java/com/jerboa/ui/components/common/AppBars.kt

```diff
@@ -642,13 +643,41 @@ fun LoadingBar(
     )
 }
 
+/**
+ * M3 doesn't have a built-in way to do this yet, so we have to do it ourselves.
+ *
+ * Supports M3 theming
+ */
+@OptIn(ExperimentalMaterialApi::class)
+@Composable
+fun JerboaPullRefreshIndicator(
+    refreshing: Boolean,
+    state: PullRefreshState,
+    modifier: Modifier = Modifier,
+    backgroundColor: Color = MaterialTheme.colorScheme.surfaceColorAtElevation(6.dp),
+    contentColor: Color = MaterialTheme.colorScheme.onSurface,
+    scale: Boolean = true,
+) {
+    PullRefreshIndicator(
+        refreshing,
+        state,
+        modifier,
+        backgroundColor,
+        contentColor,
+        scale,
+    )
+}
+
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
-fun CreateSubmitHeader(
-    title: String,
-    onClickBack: () -> Unit,
-    onSubmitClick: () -> Unit,
+fun SaveTopBar(
```

Comment:
I'd change all this `Save` terminology to `Submit` . Because sometime's they're comment edits (where save makes sense), but also comment creates (where save doesn't make sense.)

Reply:
Maybe ActionTopBar? I am okay with Submit but then I will have to change the default from save to send

Reply:
That works too. Although you could still have the default as 

`submitText: Int = R.string.save`